### PR TITLE
Birth by Sleep Map Documentation

### PR DESCRIPTION
**Pull request by Truthkey**
This documentation includes every single map in Birth by Sleep, including maps that have been removed and some that not even text exists for them, with translation for those that don't have an official one.

Also reorganizes the world list to be ordered by ID.

### CODE REVIEW

**Comment by Xeeynamo** [comment]
Woah, that's an amazing work! Just a request: could you move the list of maps to `worlds.md` in the same way as we are doing with KH2? It is to keep consistency between all the docs.

File: docs/bbs/maps.md [hunk 1]
```diff
@@ -6,4 +6,405 @@ Maps in BBS consist of an [ARC](.\arc.md) file which is loaded into memory in it
...
+|  JB22   | 闇の世界：上層 | Realm of Darkness: Upper Zone  |             | Final Mix
+|  JB23   | 闇の狭間 | The Dark Margin  |             | Final Mix
+|  JB24   | 闇シンデレラ城（FM） |   |  Cinderella's Castle (FM)         | Final Mix
+|  JB30   | デスティニーアイランド（ひる） |   | Destiny Islands (Noon)     | Text only present in HD ReMiX
+|  JB31   | トラヴァース２番街 |   | Traverse Town 2nd District            | Text only present in HD ReMiX
+|  JB32   |  |   |             | Traverse Town 1st District
+|  JB33   |  |   | Destiny Islands (Afternoon)            | Text only present in HD ReMiX
+|  JB34   |  |   | Destiny Islands (Night)            | Text only present in HD ReMiX
+|  JB35   | トラヴァース３番街 |   | Traverse Town 3rd District            | Text only present in HD ReMiX
+|  JB50   | 異空の回廊 | Lanes Between  |             |
```
Comment (Xeeynamo) [inline docs/bbs/maps.md:186]: Does JB50 changes between the game versions? For Vanilla JP it seems to be Lanes Between

Comment (Truthkey) [reply docs/bbs/maps.md:186]: JB50 has always been Lanes Between as is mentioned in this list.

File: docs/bbs/worlds.md [hunk 2]
```diff
@@ -2,28 +2,429 @@
...
+|WP|12   |[Winnie the Pooh](#Winnie-the-Pooh)
 |19|13   |unused
 |20|14   |unused
 |21|15   |unused
-|22|16   |unused
\ No newline at end of file
+|22|16   |unused
+|JF|17   |[Jump Festa](#Jump-Festa)
+
+# Map List
```
Comment (Xeeynamo) [inline docs/bbs/worlds.md:31]: You might want to change from `#` to `##` as `#` should only be used at the very top as a title (markdown language specifications)

Comment (Truthkey) [reply docs/bbs/worlds.md:31]: Alright!

**Review by Xeeynamo — APPROVED** [review]
purrrfect

**Pull request merged**